services/plot-navigator/templates/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: plot-navigator
+  labels:
+    {{- include "plot-navigator.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: plot-navigator
+  template:
+    metadata:
+      labels:
+        app: plot-navigator
```

Comment:
Should use the standard selector labels.

Reply:
Done